### PR TITLE
publish when RC1 is shipped on the 14th: Update HTTP3 Sample for RC1

### PR DESCRIPTION
**Pull request by wtgodbe**
Reflects the more streamlined behavior. We should publish this when RC1 is shipped on the 14th, and reference it from https://github.com/aspnet/specs/pull/602 (CC @danroth27).

Fixes https://github.com/dotnet/aspnetcore/issues/34488

### CODE REVIEW

**Review by Tratcher — APPROVED** [review]
We'll want to update the Localhost testing section eventually.

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@wtgodbe  I wanted to mark it as draft so it doesn't accidently get merged before the 14th

**Comment by wtgodbe** [comment]
> @wtgodbe I wanted to mark it as draft so it doesn't accidently get merged before the 14th

My mistake, I didn't see that you had marked it as a draft, I thought that was the default state. Feel free to switch it back to a draft

**Comment by Rick-Anderson** [comment]
@wtgodbe  we have an https://github.com/dotnet/AspNetCore.Docs/tree/RC1 branch now  if you have anymore PR's. I'll wait for RC1 to merge this.

File: aspnetcore/fundamentals/servers/kestrel/http3.md [hunk 1]
```diff
@@ -62,13 +62,7 @@ HTTP/3 is discovered as an upgrade from HTTP/1.1 or HTTP/2 via the `alt-svc` hea
 
 ## Get started
```
Comment (Tratcher) [inline aspnetcore/fundamentals/servers/kestrel/http3.md:63]: Update the Windows requirement to include Windows Server 2022.

Comment (wtgodbe) [reply aspnetcore/fundamentals/servers/kestrel/http3.md:63]: Done

**Pull request merged**